deploy/local/dockerfiles/server.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM php:8.3.6
+
+# install laravel dependencies including composer & npm
+RUN apt-get update -y &&\
+    apt-get install -y openssl zip unzip git nodejs npm &&\
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+RUN docker-php-ext-install pdo mysqli pdo_mysql && docker-php-ext-enable pdo_mysql
```

Comment:
Yes, because of this line:

```
RUN docker-php-ext-install pdo mysqli pdo_mysql && docker-php-ext-enable pdo_mysql
```

Quite often fail with my internet connection (probably due to blocking on my ISP), so I just make it separate so I won't execute `apt-get update` every time I failed.

Just fyi, `apt-get update` take significant time to run.